auth-web/src/components/auth/SearchBusinessForm.vue

```diff
@@ -0,0 +1,111 @@
+<template>
+    <v-container>
+      <div class="view-container">
+        <article>
+          <h1>Search Co-operatives</h1>
+          <p class="intro-text">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aliquam at porttitor sem. Aliquam erat volutpat. Donec placerat nisl magna, et faucibus arcu condimentum sed.</p>
```

Comment:
Is the text here yet to be finalized?

Reply:
Yes, I did unit test after I changed function name.

The text you mentioned is temporary. Should I remove it and just keep it empty there?

Reply:
Sorry, I was confused what you were talking about unit test. I will add unit test for StaffSearchForm component soon. Thanks.

Reply:
> Yes, I did unit test after I changed function name.
> 
> The text you mentioned is temporary. Should I remove it and just keep it empty there?

Either way is fine.